small-talk/db/registerToDB.js

```diff
@@ -1,14 +1,17 @@
 "use server"
 const uri = "mongodb+srv://smt_root:pokemonwithguns@smalltalkcluster0.jo4jne6.mongodb.net/?retryWrites=true&w=majority"
 import Mongoboi from "./mongo"
+import bcrypt from "bcryptjs";
 export default async function handleRegister(username, password, firstname, lastname, dob){ // function that sends the credentials to the database
   const mongoboi = new Mongoboi(uri, "Users");
   const date = new Date(dob).getTime() / 1000; // return the inputted date as a Unix timestamp
   await mongoboi.connect();
 
+  const hashedpass = await bcrypt.hash(password, 10);
+
   const newUser = { // schema for holding the values into certain fields for database organization
     username: username,
-    password: password,
+    password: hashedpass,
     firstname: firstname,
     lastname: lastname,
     dob: date,
```

Comment:
Doesn't have to be updated on this PR, but should consider adding in data validation of the user data, as it is it seems that the only validation so far is converting the data of birth to Unix.

Reply:
Thank you for pointing this out. This is definitely something that should be addressed. We will create a new issue to handle this